doc/api/https.md

```diff
@@ -7,47 +7,53 @@ separate module.
 
 ## Class: https.Agent
 <!-- YAML
-added: v0.4.5
+added: v0.4.4
 -->
 
-An Agent object for HTTPS similar to [`http.Agent`][].  See [`https.request()`][]
-for more information.
+**Inherits:** [`http.Agent`]
+
+An Agent object for HTTPS.
 
 ## Class: https.Server
 <!-- YAML
 added: v0.3.4
 -->
 
-This class is a subclass of `tls.Server` and emits events same as
-[`http.Server`][]. See [`http.Server`][] for more information.
+**Inherits:** [`tls.Server`]
+**Implements:** [`http.Server`]
+
+`https.Server` includes the following events, methods, and properties:
 
 ### server.setTimeout(msecs, callback)
 <!-- YAML
 added: v0.11.2
 -->
 
-See [`http.Server#setTimeout()`][].
+See [`http.Server#setTimeout()`].
 
 ### server.timeout
 <!-- YAML
 added: v0.11.2
 -->
 
-See [`http.Server#timeout`][].
+See [`http.Server#timeout`].
 
-## https.createServer(options[, requestListener])
+## https.createServer(options[, callback])
 <!-- YAML
 added: v0.3.4
 -->
 
-Returns a new HTTPS web server object. The `options` is similar to
-[`tls.createServer()`][].  The `requestListener` is a function which is
-automatically added to the `'request'` event.
+* `options` {Object} This object is used to configure the underlying [`tls.Server`].
+  See [`tls.createServer()`] for valid properties.
+* `callback` {Function} If supplied, `callback` is automatically added as a
+  `request` event handler
```

Comment:
Fair enough, I think the reason I didn't link it the first time around was because it might cause confusion linking to the `http.md` documentation, but I guess it wouldn't be that bad.